impeller/aiks/picture.cc

```diff
@@ -52,7 +52,7 @@ std::shared_ptr<Texture> Picture::RenderToTexture(
 
   // This texture isn't host visible, but we might want to add host visible
   // features to Image someday.
-  auto target = RenderTarget::CreateOffscreen(*context.GetContext(), size);
+  auto target = RenderTarget::CreateOffscreenMSAA(*context.GetContext(), size);
```

Comment:
This needs a `context->SupportsOffscreenMSAA()` guard for GLES2.

Reply:
Hmm, we should add a debug assert for `SupportsOffscreenMSAA` in `CreateOffscreenMSAA` to make this easier to catch.

Reply:
Does that mean for GLES2 we can't get anti-aliased textures?

Reply:
(and done). Maybe this will be ok because most phones will actually be on Vulkan...

Reply:
Yeah MSAA only works for the onscreen framebuffer in GLES2, so we'd need to employ other techniques if aiming for full GLES2 support in the future.